util/regexp-assemble/data/933200.data

```diff
@@ -0,0 +1,24 @@
+##! Please refer to the documentation at 
+##! https://coreruleset.org/docs/development/regexp_assemble/.
+
+##! PHP Wrappers
+##!
+##! PHP comes with many built-in wrappers for various URL-style protocols for
+##! use with the filesystem functions such as fopen(), copy(), file_exists() and
+##! filesize(). Abusing of PHP wrappers like phar:// could lead to RCE as
+##! describled by Sam Thomas at BlackHat USA 2018 (https://bit.ly/2yaKV5X), even
+##! wrappers like zlib://, glob://, rar://, zip://, etc... could lead to LFI and
+##! expect:// to RCE.
+
```

Comment:
Can you add to this comment where the wrappers come from? https://www.php.net/manual/en/wrappers.php

Reply:
Ah, yes, sure. Also added to the rule file.